src/client/_clientwindow.py

```diff
@@ -870,12 +874,25 @@ def saveWindow(self):
         util.settings.setValue("geometry", self.saveGeometry())
         util.settings.endGroup()
 
+    def show_autojoin_settings_dialog(self):
+        autojoin_channels_list = Settings.get('chat/auto_join_channels')
```

Comment:
Add '[]' as a default here, since otherwise we get None.

Reply:
ok, done, it is [] as default now